src/fixes/quote_character.rs

```diff
@@ -0,0 +1,94 @@
+use super::Fix;
+use crate::common::*;
+
+pub(crate) struct QuoteCharacterFixer<'a> {
+    name: &'a str,
+}
+
+impl Default for QuoteCharacterFixer<'_> {
+    fn default() -> Self {
+        Self {
+            name: "QuoteCharacter",
+        }
+    }
+}
+
+impl Fix for QuoteCharacterFixer<'_> {
+    fn name(&self) -> &str {
+        self.name
+    }
+
+    fn fix_line(&self, line: &mut LineEntry) -> Option<()> {
+        let value = line.get_value()?;
+        let stripped = value.strip_prefix("\"").unwrap_or(&*value);
+        let stripped = stripped.strip_suffix("\"").unwrap_or(&*value);
```

Comment:
What do you think about use `.replace()` instead of `.strip_suffix() / .strip_prefix()`
`QuoteCharacterChecker` creates warning when value contains quotes. 
I think it will be correct to delete them completely.
```suggestion
        let pure_val = val.replace("\"","")
            .replace("'","");
```

Reply:
That did not seem prudent, since the original issue is about quotes _surrounding_ value and `replace` will remove them regardless of their positioning. My opinion is that sudden removal of quotes inside the value will violate the principle of least astonishment and may actually be detrimental if user expects them to be there.

What do you think?

Reply:
It just seems to me that the `Fixer` should match the `Check`.

For example, suppose this variant of the value will also be processed incorrectly if you leave them inside:
```env
VARIABLE=text"incorrect"
```
After all, we will get
```env
VARIABLE=text"incorrect
```

Which also does not seem logical :thinking:

For similar cases we have ability to skip checks and we are already working on control comments (#237)

Reply:
@dotenv-linter/core what do you say about this?
Should we check / fix only the wrapped values or all quotes (including inside)?

Reply:
This can be fixed by only removing suffix which is a reversed version of the prefix. But I see your point :) I will wait on the final decision by the team.

Reply:
> It just seems to me that the `Fixer` should match the `Check`.

I completely agree with that.

> Should we check / fix only the wrapped values or all quotes (including inside)?

I think we should remove all quotes (including inside). This fixer should fix all warnings from `QuoteCharacterChecker`.

Reply:
@mgrachev Good, then should I also fix the wording on the `QuoteCharacterChecker`? Currently, the warning is stated as `"The value is wrapped in quotes"`, which is inconsistent with the code of the check itself.

Reply:
Yes, please 👍